packages/data_taster/data_taster.gemspec

```diff
@@ -30,7 +30,17 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{\Aexe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "license_finder", "~> 7.0"
+  spec.add_dependency "rails", "6.0.6.1"
```

Comment:
This has to be flexed to support a range of versions

Reply:
https://github.com/powerhome/power-tools/pull/214/commits/8cb6a952640d8ae38cf9fbe4676c06bf83acb359